lib/motsu-proc/tests/default.rs

```diff
@@ -0,0 +1,94 @@
+mod erc20 {
+    use alloy_primitives::U256;
+    use stylus_sdk::stylus_proc::sol_storage;
+    sol_storage! {
+        #[derive(motsu_proc::DefaultStorageLayout)]
+        pub struct Erc20 {
+            mapping(address => uint256) _balances;
+            mapping(address => mapping(address => uint256)) _allowances;
+            uint256 _total_supply;
+        }
+    }
+
+    #[motsu::test]
+    fn erc20_initiates_correctly(contract: Erc20) {
```

Comment:
nit: I'd prefer if we didn't specify `Erc20` here, but instead just the types: `mapping`, arrays, etc. The rationale is that `motsu` shouldn't concern itself with that + we already test that elsewhere.

Reply:
+1

Reply:
Agreed. I just updated it to be `mapping_initializes_and_updates`. With this we already have tests for mappings, uints(sorted and unsorted) and addresses. Does it make sense to try to cover all types? I feel pretty confortable with this coverage + the confidence we get from using it in the unit tests. But curious to hear your thoughts @bidzyyys @alexfertel